src/test/regress/output/multi_alter_table_statements.source

```diff
@@ -412,6 +412,126 @@ ERROR:  renaming distributed tables or their objects is currently unsupported
+DEBUG:  sent PREPARE TRANSACTION over connection 220007
+DEBUG:  sent PREPARE TRANSACTION over connection 220002
+DEBUG:  sent PREPARE TRANSACTION over connection 220002
+DEBUG:  sent PREPARE TRANSACTION over connection 220004
+DEBUG:  sent PREPARE TRANSACTION over connection 220004
+DEBUG:  sent PREPARE TRANSACTION over connection 220000
+DEBUG:  sent PREPARE TRANSACTION over connection 220000
+DEBUG:  sent PREPARE TRANSACTION over connection 220008
+DEBUG:  sent PREPARE TRANSACTION over connection 220008
+DEBUG:  sent PREPARE TRANSACTION over connection 220009
+DEBUG:  sent PREPARE TRANSACTION over connection 220009
+DEBUG:  sent PREPARE TRANSACTION over connection 220001
+DEBUG:  sent PREPARE TRANSACTION over connection 220001
+DEBUG:  sent PREPARE TRANSACTION over connection 220005
+DEBUG:  sent PREPARE TRANSACTION over connection 220005
+DEBUG:  sent PREPARE TRANSACTION over connection 220003
+DEBUG:  sent PREPARE TRANSACTION over connection 220003
+DEBUG:  sent COMMIT PREPARED over connection 220007
+DEBUG:  sent COMMIT PREPARED over connection 220007
+DEBUG:  sent COMMIT PREPARED over connection 220002
+DEBUG:  sent COMMIT PREPARED over connection 220002
+DEBUG:  sent COMMIT PREPARED over connection 220004
+DEBUG:  sent COMMIT PREPARED over connection 220004
+DEBUG:  sent COMMIT PREPARED over connection 220000
+DEBUG:  sent COMMIT PREPARED over connection 220000
+DEBUG:  sent COMMIT PREPARED over connection 220008
+DEBUG:  sent COMMIT PREPARED over connection 220008
+DEBUG:  sent COMMIT PREPARED over connection 220009
+DEBUG:  sent COMMIT PREPARED over connection 220009
+DEBUG:  sent COMMIT PREPARED over connection 220001
+DEBUG:  sent COMMIT PREPARED over connection 220001
+DEBUG:  sent COMMIT PREPARED over connection 220005
+DEBUG:  sent COMMIT PREPARED over connection 220005
+DEBUG:  sent COMMIT PREPARED over connection 220003
+DEBUG:  sent COMMIT PREPARED over connection 220003
+RESET client_min_messages;
+DROP INDEX temp_index_3;
+-- verify that citus.multi_shard_commit_protocol value is not changed
+SHOW citus.multi_shard_commit_protocol;
+ citus.multi_shard_commit_protocol 
+-----------------------------------
+ 1pc
+(1 row)
+
+-- verify that not any of shard placements are marked as failed when a query failure occurs
+CREATE TABLE test_ab (a int, b int);
+SELECT master_create_distributed_table('test_ab', 'a', 'hash');
+ master_create_distributed_table 
+---------------------------------
+ 
+(1 row)
+
+SELECT master_create_worker_shards('test_ab', 8, 2);
+ master_create_worker_shards 
+-----------------------------
+ 
+(1 row)
+
+INSERT INTO test_ab VALUES (2, 10);
+INSERT INTO test_ab VALUES (2, 11);
+CREATE UNIQUE INDEX temp_unique_index_1 ON test_ab(a);
+WARNING:  could not create unique index "temp_unique_index_1_220016"
+DETAIL:  Key (a)=(2) is duplicated.
+CONTEXT:  while executing command on localhost:57638
+ERROR:  could not execute DDL command on worker node shards
+SELECT * FROM pg_dist_shard_placement WHERE shardstate=3;
```

Comment:
This query seems like it would break if any other tests produce shardstate 3 in the future.